svc-events/events/submittestevent.go

```diff
@@ -203,7 +203,9 @@ func validAndGenSubTestReq(reqBody []byte) (*ForwardEvent, string, string, []int
 	if val, ok := req["OriginOfCondition"]; ok {
 		switch v := val.(type) {
 		case string:
-			testEvent.OriginOfCondition = v
+			testEvent.OriginOfCondition = &common.Link{
```

Comment:
Add comment "As per EventService spec in the SubmitTestEvent subschema OriginOfCondition is a string. However we need to convert this to an object as the event publisher will drop these events.

Reply:
added comment